tests/Integrations/Lumen/V5_2/CommonScenariosTest.php

```diff
@@ -109,7 +111,10 @@ public function provideSpecs()
                             'lumen_test_app',
                             'web',
                             'Laravel\Lumen\Application.handleFoundRoute'
-                        )->withExistingTagsNames([
+                        )->withExactTags([
+                            'lumen.route.action' => 'App\Http\Controllers\ExampleController@error',
```

Comment:
`error` route has no name set, and to be honest I think we should leave as it is rather than adding a name so we add some diversity to use cases we test